src/test/java/spark/utils/StringUtilsTest.java

```diff
@@ -0,0 +1,152 @@
+	}
+
+    @Test
+    public void testCleanPath() {
+        Assert.assertNull(StringUtils.cleanPath(null));
+        Assert.assertEquals("foo:/../Bar", StringUtils.cleanPath("foo:/foo/../../Bar"));
+    }
+
+	@Test
+	public void testCollectionToDelimitedString() {
+        Assert.assertEquals("",
+            StringUtils.collectionToDelimitedString(new ArrayList(), null, null, null));
+
+		ArrayList coll = new ArrayList();
+		coll.add(10);
+
+		Assert.assertEquals("10!!!!",
+            StringUtils.collectionToDelimitedString(coll, null, "", "!!!!"));
+
+        coll.add(null);
+        coll.add(-2_113_912_831);
+
+        Assert.assertEquals("!10?\\!null?\\!-2113912831?",
+            StringUtils.collectionToDelimitedString(coll, "\\", "!", "?"));
+        Assert.assertEquals("",
+            StringUtils.collectionToDelimitedString(null, null));
+	}
+
+	@Test
+	public void testDeleteAny() {
+		Assert.assertEquals("#", StringUtils.deleteAny("#", "!!!!!!!!"));
+        Assert.assertEquals("", StringUtils.deleteAny("#", "\"\"\"#!!!!"));
+        Assert.assertEquals("\"", StringUtils.deleteAny("\"", null));
+
+        Assert.assertNull(StringUtils.deleteAny(null, null));
+	}
+
+    @Test
+    public void testDelimitedListToStringArray() {
+        Assert.assertArrayEquals(new String[] {},
+            StringUtils.delimitedListToStringArray(null, null, null));
+        Assert.assertArrayEquals(new String[] { "fooBar" },
+            StringUtils.delimitedListToStringArray("fooBar", null, null));
+        Assert.assertArrayEquals(new String[] { "foo" , ""},
+            StringUtils.delimitedListToStringArray("fooBar", "a", "Bar"));
+        Assert.assertArrayEquals(new String[] { "" , "Bar"},
+            StringUtils.delimitedListToStringArray("fooBar", "foo", ""));
+        Assert.assertArrayEquals(new String[] { "f", "o", "o", "B", "a", "r"},
+            StringUtils.delimitedListToStringArray("fooBar", "", ""));
+    }
+
+    @Test
+    public void testDelimitedListToStringArray2() {
+        Assert.assertArrayEquals(new String[] { "" , "Bar"},
+            StringUtils.delimitedListToStringArray("fooBar", "foo"));
+    }
+
+	@Test
+	public void testGetFilename() {
+		Assert.assertEquals("", StringUtils.getFilename(""));
+        Assert.assertEquals("", StringUtils.getFilename("///////"));
+
+        Assert.assertNull(StringUtils.getFilename(null));
+	}
+
+	@Test
+	public void testHasLength() {
+		Assert.assertFalse(StringUtils.hasLength(""));
+        Assert.assertFalse(StringUtils.hasLength(null));
+
+        Assert.assertTrue(StringUtils.hasLength("00000000"));
+	}
+
+
+	@Test
+	public void testIsBlank() {
+		Assert.assertFalse(StringUtils.isBlank("@@@@@@@@"));
+
+        Assert.assertTrue(StringUtils.isBlank(""));
+        Assert.assertTrue(StringUtils.isBlank(" "));
+        Assert.assertTrue(StringUtils.isBlank(null));
+	}
+
+	@Test
+	public void testIsEmpty() {
+        Assert.assertFalse(StringUtils.isEmpty(0));
```

Comment:
Shouldn't we be passing a string? Admittedly, the code returns true in this case.

Reply:
Could also test for `""` for completeness.